toolchest_client/tools.py

```diff
@@ -40,9 +40,9 @@ def cutadapt(cutadapt_args, **kwargs):
 
     Usage::
 
-        >>> import toolchest_client as tc
-        >>> tc.cutadapt(
-        ...     "-a AACCGGTT",
+        >>> import toolchest_client as toolchest
+        >>> toolchest.cutadapt(
+        ...     tool_args="-a AACCGGTT",
```

Comment:
Specifically calling out this change – what are you thoughts?

Reply:
I'm okay with this. It's less concise but more explicit.